### PR TITLE
feat: apply real data and func at workspaceInfo

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA
image

### CODE REVIEW

File: apps/web/src/services/workspace-home/components/WorkspaceInfo.vue [hunk 1]
```diff
@@ -1,32 +1,122 @@
...
+    getCurrentRoleInfo: computed<UserState>(() => store.getters['user/getCurrentRoleInfo']),
     hasAdminOrWorkspaceOwnerRole: computed<boolean>(() => store.getters['user/hasAdminOrWorkspaceOwnerRole']),
     currentWorkspace: computed<WorkspaceModel|undefined>(() => userWorkspaceGetters.currentWorkspace),
     workspaceList: computed<WorkspaceModel[]>(() => userWorkspaceGetters.workspaceList),
 });
 const state = reactive({
+    isWorkspaceOwner: computed<boolean>(() => storeState.getCurrentRoleInfo.roleType === ROLE_TYPE.WORKSPACE_OWNER),
     selectedWorkspace: computed<WorkspaceModel>(() => storeState.workspaceList.find((workspace) => workspace.workspace_id === storeState.currentWorkspace?.workspace_id) || {} as WorkspaceModel),
+    workspaceUserCnt: undefined as number|undefined,
+    appsCnt: undefined as number|undefined,
```
Comment (yuda110) [inline apps/web/src/services/workspace-home/components/WorkspaceInfo.vue:61]: nitpick: It's hard to know what 'cnt' means!😅

**Review by yuda110 — APPROVED** [review]
LGTM!! I left some comments!

File: apps/web/src/services/workspace-home/components/WorkspaceInfo.vue (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/services/workspace-home/components/WorkspaceInfo.vue:61]: I think so too~

**Review by piggggggggy — APPROVED** [review]
LGTM~

File: apps/web/src/services/workspace-home/components/WorkspaceInfo.vue [hunk 2]
```diff
@@ -1,32 +1,122 @@
...
+});
+
+const handleClickInviteButton = () => {
+    router.push({ name: IAM_ROUTE.USER._NAME });
```
[the remark this answers is not in the document]
Comment (skdud4659) [reply apps/web/src/services/workspace-home/components/WorkspaceInfo.vue:65]: Not needed here!

**Pull request merged**